ci/rtools/README.md

```diff
@@ -0,0 +1,16 @@
+# rtools40 patches for AWS SDK and related libs
```

Comment:
This lacks a license header and is flagged by the Lint CI step.

Reply:
Added license header